src/components/PlayerSurface.tsx

```diff
@@ -45,7 +45,7 @@ export interface PlayerProps extends BaseComponentProps {
    */
   playsInline?: boolean
   /**
-   * Enable focus-based keyboard interactivity. Default: true.
+   * Enable focus-based keyboard interactivity. Default: false.
```

Comment:
This auto-focus logic is for smart TVs but on web it's also on and it causes the page to scroll the player into view a few seconds after interaction which can be really annoying on pages with a scrollbar. So I'm disabling this by default.